src/main/java/mods/eln/node/six/SixNodeBlock.java

```diff
@@ -123,7 +123,7 @@ public int getRenderType() {
 	 * return 255; }
 	 */
 
-    @Override
+
```

Comment:
Removing an @Override does *not* mean the code is fine. It means the code was doing something before, and now it's no longer doing it. You need to figure out what, then do the new equivalent, or else leave it alone.